pkg/middlewares/compress/acceptencoding.go

```diff
@@ -0,0 +1,137 @@
+package compress
+
+import (
+	"slices"
+	"strconv"
+	"strings"
+)
+
+const acceptEncodingHeader = "Accept-Encoding"
+
+const (
+	brotliName    = "br"
+	gzipName      = "gzip"
+	identityName  = "identity"
+	wildcardName  = "*"
+	notAcceptable = "not_acceptable"
```

Comment:
Couldn't we get rid of `notAcceptable` and only use identity?

Reply:
The spec defines "not acceptable".
In the future, this value should be used to generate 415.

Reply:
Ah, right!